ctapipe_io_nectarcam/__init__.py

```diff
@@ -328,54 +335,52 @@ def fill_nectarcam_service_container_from_zfile(self):
         # svc_container.pre_proc_algorithms = camera_config.nectarcam.pre_proc_algorithms
 
     def fill_nectarcam_event_container_from_zfile(self, event):
-        event_container = self.data.nectarcam.tel[self.camera_config.telescope_id].evt
-
-        event_container.configuration_id = event.configuration_id
-        event_container.event_id = event.event_id
-        event_container.tel_event_id = event.tel_event_id
-        event_container.pixel_status = event.pixel_status
-        event_container.ped_id = event.ped_id
-        event_container.module_status = event.nectarcam.module_status
-        event_container.extdevices_presence = event.nectarcam.extdevices_presence
-        #event_container.tib_data = event.nectarcam.tib_data
-        #event_container.cdts_data = event.nectarcam.cdts_data
-        event_container.swat_data = event.nectarcam.swat_data
-        event_container.counters = event.nectarcam.counters
+        nec_evt = self.data.nectarcam.tel[self.camera_config.telescope_id].evt
+
+        nec_evt.configuration_id = event.configuration_id
+        nec_evt.event_id = event.event_id
+        nec_evt.tel_event_id = event.tel_event_id
+        nec_evt.pixel_status = event.pixel_status
+        nec_evt.ped_id = event.ped_id
+        nec_evt.module_status = event.nectarcam.module_status
+        nec_evt.extdevices_presence = event.nectarcam.extdevices_presence
+        #nec_evt.tib_data = event.nectarcam.tib_data
```

Comment:
Why are those two not included?

Reply:
Since TIB and CDTS data are decoded, I thought there was no need to keep the original binary data.